src/Zenodo.ts

```diff
@@ -0,0 +1,60 @@
+import { Deposition } from './depositions/Deposition';
+import type { ListDepositionsOptions } from './depositions/ListDepositionsOptions';
+import type { DepositionMetadata } from './depositions/types';
+import { fetchZenodo } from './fetchZenodo';
+
+export class Zenodo {
+  host: string;
+  accessToken: string;
+  baseURL: string;
+
+  constructor(options) {
+    const { accessToken, host = 'sandbox.zenodo.org' } = options;
```

Comment:
I would either put the production env by default or specify it in the example on the readme.